images/openldap/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM openshift/openldap-2441-centos7
```

Comment:
Why isn't the version a tag?  There was probably a reason, but it's not obvious to me.  Are we having tags for our openldap image follow openshift versions for some reason?